doc/io-properties-file.md

```diff
@@ -24,6 +24,12 @@ Additionally the following optional properties can be added:
 
 * `read_saturation_length`: read buffer length to saturate the device throughput
 * `write_saturation_length`: write buffer length to saturate the device throughput
+* `min_iop_size`: The minimum IOP size the application will issue.
+    Should match (or be smaller) than IOP size used in the IOPS test.
+    Used in the io-queue cost model.
+* `max_iop_size`: The maximum IOP size the application will issue.
```

Comment:
Will or should? Currently max io size is advertised via  io_queue::get_request_limits() and it doesn't look like the values returned from there are anyhow affected by that io-properties parameter, are they?

Reply:
I see. Well yes, I agree that what iotune uses vs what it adverises up the stack should not necessarily match. Provided [my understanding](https://github.com/scylladb/seastar/pull/2801#discussion_r2179938259) is (still) correct, I would expect values and comments that describe that new "line on iops;bandwidth diskplorer plane"